staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time.go

```diff
@@ -180,3 +180,12 @@ func (t Time) MarshalQueryParameter() (string, error) {
 
 	return t.UTC().Format(time.RFC3339), nil
 }
+
+// String returns the string representation of the wrapped time.Time
+func (t *Time) String() string {
```

Comment:
this was previously reported in https://github.com/kubernetes/kubernetes/issues/105981 but closed because of https://github.com/kubernetes/kubernetes/issues/105981#issuecomment-963573312

almost identical fixes were previously proposed in https://github.com/kubernetes/kubernetes/pull/108436 and https://github.com/kubernetes/kubernetes/pull/105982 and declined because of https://github.com/kubernetes/kubernetes/pull/105982#discussion_r742312742 and https://github.com/kubernetes/kubernetes/issues/105981#issuecomment-963573312

Reply:
Looks like this can't be fixed, but people will keep trying...

@kaisoz: can you change this PR so that instead of adding a `String` method, a comment gets added why there is no String implementation ("historic reasons") and why it cannot be added now (see https://github.com/kubernetes/kubernetes/issues/105981#issuecomment-963573312)?